cmf-cli/resources/template_feed/init/.template.config/template.json

```diff
@@ -486,6 +486,12 @@
       "defaultValue": "",
       "replaces": "<%= $CLI_PARAM_AppName %>"
     },
+    "appNameMin": {
```

Comment:
this is a parameter but doesn't seem to be filled in a command, should this be a derived symbol from appName?

Reply:
It was a rebase error, a parameter should be created on Init command